magento/pwa-studio
# [Doc] Update 4.0.0 release notes

## Description

Updates the 4.0.0 release notes with more up to date content

## Related Issue

Closes #1647 

## Verification Steps
1. Open `CHANGELOG.md`
2. Verify it contains 4.0.0 release note items
